src/commands/deploy.ts

```diff
@@ -181,6 +181,12 @@ export async function deployClasses(configuration: YamlProjectConfiguration, clo
     sdkResponse
   );
 
+  const classesWithEmptyMethods = checkClassesMethods(projectConfiguration.classes);
```

Comment:
Let's better name the variable and the function name.
Variable could be changed to 'classesWithNoMethods'.

Function name could be renamed to 'getNoMethodClasses()'